src/platform/Kconfig

```diff
@@ -453,6 +452,39 @@ config RIMAGE_SIGNING_SCHEMA
 	help
 	  Signing schema name used by rimage to decide how to build final binary
 
+config FW_NAME
+	string "Firmware name"
+	default "byt" if BAYTRAIL
+	default "cht" if CHERRYTRAIL
+	default "hsw" if HASWELL
+	default "bdw" if BROADWELL
+	default "apl" if APOLLOLAKE
+	default "cnl" if CANNONLAKE
+	default "sue" if SUECREEK
+	default "icl" if ICELAKE
+	default "tgl" if TIGERLAKE
+	default "imx8" if IMX8
+	default "imx8x" if IMX8X
+	default "imx8m" if IMX8M
+	default "imx8ulp" if IMX8ULP
+	default "rn" if RENOIR
+	default "mt8195" if MT8195
+	default ""
+	help
+	Every file generated by the build system will start with this name plus some extension.
+
+config CAVS_MANIFEST
```

Comment:
This should be based on number, where we could do things like ```if SOF_MANIFEST == 2``` and it should be set in the Kconfig as a menuchoice (as IIRC this would make it mutually exclusive to other manifest versions).